### DISCUSSION TITLE
Fetch Upstream button is hard to see in dark mode.

### DISCUSSION THREAD

**Discussion by fps7806** [community/community#15411] [Accessibility]
Took me a while to find it:
![image](https://user-images.githubusercontent.com/7218057/164766301-dad3fe9d-55d6-4dd8-8051-6f6b184f81bb.png)

**Comment by cehfisher** [comment]
Hello @fps7806 - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you!

**Comment by cehfisher** [comment]
Adding image of current dark theme merge buttons. It looks like the original issue has been resolved.

![dark-button-update](https://github.com/user-attachments/assets/5fa1efb9-4711-4b5d-b0b6-9101ff00361c)
